bot.py

```diff
@@ -8,6 +8,7 @@
 import asyncio
 import queue
 import traceback
+from time import time
```

Comment:
I would prefer 'import time' and then do time.time(). Otherwise you cannot use anything else from the time package.

Reply:
done